### PR TITLE
Add changelog for 4.1.0

### PR DESCRIPTION
**Pull request by bernd**

### CODE REVIEW

File: pages/enterprise/changelog.rst [hunk 1]
```diff
@@ -2,6 +2,44 @@
...
 
+Graylog Enterprise 4.1.0
+========================
+
+Released: 2021-06-23
+
+Enterprise
+----------
+
+**Added**
```
Comment (linuspahl) [inline pages/enterprise/changelog.rst:13]: We can also mention https://github.com/Graylog2/graylog-plugin-enterprise/pull/2024 here.

File: pages/changelog.rst [hunk 2]
```diff
@@ -2,6 +2,91 @@
...
+- Add ``allowed_auxiliary_paths`` config option to optionally restrict file system access for the CSV File data dapter. `Graylog2/graylog2-server#10244 <https://github.com/Graylog2/graylog2-server/issues/10244>`_
+- Add timerange indicator to widgets. `Graylog2/graylog2-server#7358 <https://github.com/Graylog2/graylog2-server/issues/7358>`_ `Graylog2/graylog2-server#10508 <https://github.com/Graylog2/graylog2-server/issues/10508>`_
+- Allow filtering for entity ID in pagination. `Graylog2/graylog2-server#10537 <https://github.com/Graylog2/graylog2-server/issues/10537>`_
+- Add built-in Prometheus exporter. `Graylog2/graylog2-server#10767 <https://github.com/Graylog2/graylog2-server/issues/10767>`_
```
Comment (linuspahl) [inline pages/changelog.rst:29]: We can also mention https://github.com/Graylog2/graylog2-server/pull/9836 here. The title could be: "Add color scale configuration options for heatmap widget."

File: pages/enterprise/changelog.rst (hunk 1 above)
Comment (bernd) [reply pages/enterprise/changelog.rst:13]: @linuspahl Thank you. Added!

File: pages/changelog.rst (hunk 2 above)
Comment (bernd) [reply pages/changelog.rst:29]: Done, thanks!

**Review by waab76 — DISMISSED** [review]
I don't see anything missing (mostly looking at Integrations, Enterprise, and Enterprise Integrations)

File: pages/enterprise/changelog.rst [hunk 3]
```diff
@@ -2,6 +2,45 @@
...
+
+- Add theme customization options to allow the usage of custom colors.
+- Add support for global notifications to display announcements and other messages to all users or a selected group of users.
+- Add authentication and team-sync support for the Okta indentity provider.
+- Add support for the Graylog Forwarder. The Graylog Forwarder is a standalone agent for sending send log data to Graylog Cloud or an on-premise Graylog Server cluster.
```
Comment (mpfz0r) [inline pages/enterprise/changelog.rst]:
```suggestion
- Add support for the Graylog Forwarder. The Graylog Forwarder is a standalone agent for sending log data to Graylog Cloud or an on-premise Graylog Server cluster.
```

Comment (bernd) [reply pages/enterprise/changelog.rst]: Thanks!

**Review by malcyon — APPROVED** [review]
Looks good!

**Pull request merged**